tuf_conformance/test_updater_delegation_graphs.py

```diff
@@ -0,0 +1,151 @@
+from dataclasses import astuple, dataclass, field
+from typing import Any
+
+import pytest
+from tuf.api.metadata import (
+    SPECIFICATION_VERSION,
+    TOP_LEVEL_ROLE_NAMES,
+    DelegatedRole,
+    TargetFile,
+    Targets,
+)
+
+from tuf_conformance.client_runner import ClientRunner
+from tuf_conformance.repository_simulator import RepositorySimulator
+from tuf_conformance.simulator_server import SimulatorServer
+
+# DataSet is only here so type hints can be used.
+DataSet = dict[str, Any]
+
+
+@dataclass
+class DelegationTester:
+    delegator: str
+    rolename: str
+    keyids: list[str] = field(default_factory=list)
+    threshold: int = 1
+    terminating: bool = False
+    paths: list[str] | None = field(default_factory=lambda: ["*"])
+    path_hash_prefixes: list[str] | None = None
+
+
+@dataclass
+class TestTarget:
```

Comment:
```
test_updater_delegation_graphs.py:32: PytestCollectionWarning: cannot collect test class 'TestTarget' because it has a __init__ constructor (from: tuf_conformance/test_updater_delegation_graphs.py)
    @dataclass
```

Reply:
Renamed to `TargetTest`